tools/statsnoop.py

```diff
@@ -56,42 +56,41 @@
     char fname[NAME_MAX];
 };
 
-BPF_HASH(args_filename, u32, const char *);
-BPF_HASH(infotmp, u32, struct val_t);
+BPF_HASH(infotmp, u64, struct val_t);
 BPF_PERF_OUTPUT(events);
 
 int syscall__entry(struct pt_regs *ctx, const char __user *filename)
 {
     struct val_t val = {};
-    u32 pid = bpf_get_current_pid_tgid();
+    u64 pid_tgid = bpf_get_current_pid_tgid();
+    u32 pid = pid_tgid >> 32;
```

Comment:
Using `tid = bpf_get_current_pid_tgid();` totally fine as the key although using pid_tgid works too. Let us just maintain the original code but rename "pid" to "tid". In some other bcc tools, people use tid too. Let us try to get consistent about this. Something like
```
  u64 pid_tgid = bpf_get_current_pid_tgid();
  u32 pid = pid_tgid >> 32, tid = (u32)pid_tgid;
```

Reply:
Ah, tid is unique across all processes, right ?